src/profile/components/deleteProfile/DeleteProfile.tsx

```diff
@@ -87,8 +87,11 @@ function DeleteProfile(props: Props) {
 
         <Checkbox
           onChange={handleDeleteInstructions}
+          id="deleteInstructions"
           name="deleteInstructions"
-          label={
+          // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+          // @ts-ignore
```

Comment:
Do we have an issue or a plan for how we can remove the need for these ts-ignores?

Reply:
I've opened an issue about this in HDS.